charts/ai-software-templates/chatbot/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 kubeVersion: ">= 1.27.0-0"
 annotations:
-  charts.openshift.io/name: Chatbot AI Software Template
-description: A helm chart for the AI Software Templates. For more information please check https://github.com/redhat-ai-dev/ai-lab-helm-charts.git
-name: chatbot-ai-software-template
+  charts.openshift.io/name: Chatbot AI Sample
+description: A helm chart for the Chatbot AI Sample app. For more information please check https://github.com/redhat-ai-dev/ai-lab-helm-charts.git
+name: chatbot-ai-sample
 tags: chatbot,llama.cpp,ai-lab
 version: 0.1.0
```

Comment:
I am not 100% sure but might need a version bump here.

Reply:
I agree better to bump. We can onboard 0.1.1 there's no problem, originally I thought we might include it in the 0.1.0 but I feel is cleaner this way. Addressed in [9f48668](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/23/commits/9f48668c4c3e954ee822147a32fd550c846cea84)